build.gradle

```diff
@@ -119,6 +119,8 @@ dependencies {
     implementation fileTree(dir: sqlJarDirectory, include: ["opensearch-sql-${version}.jar", "ppl-${version}.jar", "protocol-${version}.jar"])
     compileOnly "org.opensearch:common-utils:${version}"
     compileOnly "org.jetbrains.kotlin:kotlin-stdlib:${kotlin_version}"
+    compileOnly "org.opensearch:opensearch-job-scheduler-spi:${version}"
```

Comment:
If it's `compileOnly`, who will provide the class during runtime? I don't think skills depends on job scheduler

Reply:
This is needed for AD tool due to AD's dependency. I've used the same logic as AD plugin here for consistency regarding the job scheduler dependencies.